Directory.Build.targets

```diff
@@ -87,11 +87,15 @@
       <TargetingPackVersion Condition="'$(IsServicingBuild)' != 'true'">$(MicrosoftNETCoreAppPackageVersion)</TargetingPackVersion>
     </KnownFrameworkReference>
 
+    <KnownFrameworkReference Condition="'$(UseAspNetCoreSharedRuntime)' != 'true'" Remove="Microsoft.AspNetCore.App" />
```

Comment:
Do you need the conditional check?

Reply:
Probably not, but I think people want this merged ASAP. I'll make a follow up Pr for nits.

Reply:
Sounds good.